setup.py

```diff
@@ -43,7 +43,7 @@ def get_version(package):
     packages=find_packages(exclude=["tests*"]),
     install_requires=[
         "Django>=2.1.0",
-        "django-graphql-jwt==0.3.0",
+        "django-graphql-jwt>=0.3.1,<0.4.0",
```

Comment:
I debated putting this as `django-graphql-jwt==0.3.1` since the whole reason for this PR is from a breaking change on a patch update...but also new features with no work is nice.

Reply:
`"django-graphql-jwt>=0.3.1,<0.4.0"` is perfect 👍